src/app/api/grant-application/update/route.ts

```
@@ -31,6 +31,9 @@ async function updateGrantApplication(
       data.twitter !== undefined
         ? extractSocialUsername('twitter', data.twitter) || ''
         : undefined,
+    github: !!data.github
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)